backend/tests/test_retrieve_test_view.py

```diff
@@ -63,7 +63,64 @@ def test_get_real_sample(self):
             "emibSampleTest", timezone.now(), TEST_QUESTIONS)
         expected_json = {
             "test_internal_name": "emibSampleTest",
-            "questions": []

```

Comment:
Do we really need to do these types of test? Because as soon as we'll update the content, these tests will fail.

Reply:
I've put this back to WIP... I want to look over part of it because I think I can cut down on the loops/ifs. That, and I need to re-vist how the ids are made.
However, I'm close to a small PR for updating the json to look like the desired return and updating the redux accordingly

Reply:
I'm currently using this to make sure there's not a break down for this api. I'm not planning to make it for any test other than the sample test.